README.md

```diff
@@ -1,3 +1,32 @@
 # MNXB01-learn
 
-Once upon a time, in a land closer than you think, there lived a **rich** but _cruel_ king - The Coldran of Kor. He was feared by all his subject and everyone who had the unfortunate chance to hear his name spoken.
+Once upon a time, in a land closer than you think, 
+there lived a **rich** but _cruel_ king - The Coldran of Kor. 
+He was feared by all his subject and everyone who had the unfortunate 
+chance to hear his name spoken.
```

Comment:
I don't want line 6 to be a separated from line 5. Follow up line 5 with line 6.